src/components/widgets-settings/param-selector-classic/param-selector-classic.directive.coffee

```diff
@@ -0,0 +1,60 @@
+module = angular.module('impac.components.widgets-settings.param-selector-classic', [])
```

Comment:
This directive is quite similar to the params-selector we already have: don't you think we could add a new parameter to the existing directive instead? Like:
```
scope: {
  ...
  classic: '=?'
}
```

A test on the value of $scope.classic (in the template) could determine if you should display the usual container or a simple <select>.